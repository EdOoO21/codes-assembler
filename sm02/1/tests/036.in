-881 -958 964 641 880 862 617 -418 792 -840 -1000 29 284 -639 -493 -449 495 336 -115 -254 147 -451 111 -841 179 -165 694 3 -202 -293 548 650 -190 821 -502 -198 -657 -748 52 18 436 566
