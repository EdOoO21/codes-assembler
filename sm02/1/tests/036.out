566
436
18
52
-748
-657
-198
-502
821
-190
650
548
-293
-202
3
694
-165
179
-841
111
-451
147
-254
-115
336
495
-449
-493
-639
284
29
-1000
-840
792
-418
617
862
880
641
964
-958
-881
